Dockerfile

```diff
@@ -44,21 +50,29 @@ COPY ./serving/servesink/src ./servesink/src
 COPY ./serving/extras/upstreams/src ./extras/upstreams/src
 COPY ./serving/backoff/src ./backoff/src
 COPY ./serving/numaflow-models/src ./numaflow-models/src
+COPY ./serving/source-sink/src ./source-sink/src
+COPY ./serving/source-sink/build.rs ./source-sink/build.rs
+COPY ./serving/source-sink/proto ./source-sink/proto
 
 # Build the real binaries
-RUN touch src/main.rs servesink/main.rs extras/upstreams/main.rs numaflow-models/main.rs && \
-    cargo build --release
+RUN touch src/main.rs servesink/src/main.rs numaflow-models/src/main.rs source-sink/src/main.rs && \
+    cargo build --workspace --all --release
 
 ####################################################################################################
 # numaflow
 ####################################################################################################
 ARG BASE_IMAGE
-FROM ${BASE_IMAGE} as numaflow
+FROM debian:bookworm as numaflow
```

Comment:
This is going out of control.

1. Image build time in the CI increased from `<2 mins` to `> 7 mins`
2. Image size and security vulunabilities.

Reply:
we can handle this in follow up PR, already created an action item.